build_for_sdk.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+xcodebuild ARCHS="armv7 armv7s arm64" ONLY_ACTIVE_ARCH=NO -configuration Release -project sqlcipher.xcodeproj
+xcodebuild ARCHS="x86_64 i386" ONLY_ACTIVE_ARCH=NO -configuration Release -project sqlcipher.xcodeproj -sdk iphonesimulator
+lipo -create build/Release-iphoneos/libsqlcipher.a build/Release-iphonesimulator/libsqlcipher.a -output libsqlcipher.a
```

Comment:
Typically, you'd specify the `CONFIGURATION_BUILD_DIR` variable in your build, then reference it in your subsequent commands, as `build/Release-iphoneos/` is effectively the hard-coded default value of a variable.  So it would look something like this:

```bash
ROOT_DIRECTORY=$(cd "$(dirname ${BASH_SOURCE[0]})" && pwd)
DEVICE_CBD="${ROOT_DIRECTORY}/build/device"
SIM_CBD="${ROOT_DIRECTORY}/build/sim"
xcodebuild ARCHS="armv7 armv7s arm64" ONLY_ACTIVE_ARCH=NO CONFIGURATION_BUILD_DIR="$DEVICE_CBD" -configuration Release -project sqlcipher.xcodeproj
xcodebuild ARCHS="x86_64 i386" ONLY_ACTIVE_ARCH=NO CONFIGURATION_BUILD_DIR="$SIM_CBD" -configuration Release -project sqlcipher.xcodeproj -sdk iphonesimulator
lipo -create "$DEVICE_CBD/libsqlcipher.a" "$SIM_CBD/libsqlcipher.a" -output "$ROOT_DIRECTORY/libsqlcipher.a"
```

Anyway, food for thought. :)